src/cascadia/TerminalControl/ControlCore.cpp

```diff
@@ -435,21 +449,10 @@ namespace winrt::Microsoft::Terminal::Control::implementation
         }
         else
         {
-            _connection.WriteInput(winrt_wstring_to_array_view(wstr));
+            _sendInputToConnection(wstr);
         }
     }
 
-    // Method Description:
-    // - Writes the given sequence as input to the active terminal connection,
-    // Arguments:
-    // - wstr: the string of characters to write to the terminal connection.
-    // Return Value:
-    // - <none>
-    void ControlCore::SendInput(const std::wstring_view wstr)
-    {
-        _sendInputToConnection(wstr);
-    }

```

Comment:
`SendInput` now does what `_sendInputToConnection` previously did.
`_sendInputToConnection` is now a write-directly-to-connection function.